products/rhel10/profiles/e8.profile

```diff
@@ -38,3 +38,5 @@ selections:
     - '!package_rsh_removed'
     - '!package_rsh-server_removed'
     - '!security_patches_up_to_date'
+     # the rule ensure_redhat_gpgkey_installed needs a special package on RHEL 10 which is not installed by default
+    - package_sequoia-sq_installed
```

Comment:
You have two options how to solve the failing static-checks reference test.

1. Add the rule to the control file and then deselect it in profile files in products where this shouldn't be present.
2. Add an ISM_O reference to the rule.yml file in package_sequoia-sq_installed.

Reply:
Option 2 will not work, you cannot mix control file references and in file references.